kong.conf.default

```diff
@@ -452,17 +452,32 @@
                                  # To read the file again after modifying it,
                                  # Kong must be reloaded.
 
-#dns_order = LAST,SRV,A,CNAME    # the order in which to resolve different
+#dns_order = LAST,SRV,A,CNAME    # The order in which to resolve different
                                  # record types. The `LAST` type means the
                                  # type of the last successful lookup (for the
                                  # specified name). The format is a (case
                                  # insensitive) comma separated list.
 
+#dns_stale_ttl = 4.0             # Once a record expires, how much longer (in
+                                 # seconds) may the current stale record be 
+                                 # used while the refresh query is executed in
+                                 # the background.
+                                 # So stale data will be used from expiry of a
+                                 # record until either the refresh query
+                                 # completes, or the `dns_stale_ttl` number of
+                                 # seconds have passed.
```

Comment:
This sentence reads weird because it starts with "So", which feels colloquial.
```
Stale records will still be returned when they expired,
until the refresh query completes, or the `dns_stale_ttl`
number of seconds have passed. 
```